MAUI/StepProgressBar/migration.md

```diff
@@ -0,0 +1,425 @@
+<tr>
+<td>{{'[SfStepProgressBar]()'| markdownify }}</td>
+<td>{{'[SfStepProgressBar]()'| markdownify }}</td>
+<td>Represents a class which holds the step progressbar common properties in SfStepProgressBar.</td>
+</tr> 
+<tr>
+<td>{{'[StepView ]()' | markdownify }}</td>
+<td>{{'[StepProgressBarItem]()' | markdownify }}</td>
+<td>Represents a class which is used to customize the step progress bar item.</td>
+</tr> 
+<tr>
+<td>{{'[IStepStyle]()' | markdownify }}</td>
+<td>{{'[StepSettings]()' | markdownify }}</td>
+<td>Represents a class which is used to customize the style of the step view.</td>
+</tr>
+<tr>
+<td>{{'[StepTappedEventArgs ]()' | markdownify }}</td>
+<td>{{'[StepTappedEventArgs]()' | markdownify }}</td>
+<td>Represents a class which is used to hold the step tapped event arguments.</td>
+</tr> 
+<tr>
+<td>{{'[StatusChangedEventArgs ]()' | markdownify }}</td>
+<td>{{'[StepStatusChangedEventArgs]()' | markdownify }}</td>
+<td>Represents a class which is used to hold the status changed event arguments.</td>
+</tr>
+</table>
+
+## Properties 
+
+### SfStepProgressBar
+
+The following code example, explains how to initialize the properties of the `Xamarin SfStepProgressBar` and `.NET MAUI SfStepProgressBar` class.
+
+<table>
+<tr>
+<th>Xamarin SfStepProgressBar</th>
+<th>.NET MAUI SfStepProgressBar</th></tr>
+<tr>
+<td>
+{% tabs %}
+
+{% highlight xaml %}
+
+<ContentPage
+xmlns:progressBar="clr-namespace:Syncfusion.XForms.ProgressBar;assembly=Syncfusion.SfProgressBar.XForms">
+
+    <progressBar:SfStepProgressBar />
+
+</ContentPage>
+
+{% endhighlight %}
+
+{% highlight C# %}
+
+using Syncfusion.XForms.ProgressBar;
+...
+
+SfStepProgressBar sfStepProgressBar = new SfStepProgressBar ();
+this.Content = sfStepProgressBar ;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+</td>
+<td>
+{% tabs %}
+
+{% highlight xaml %}
+
+<ContentPage
+xmlns:progressBar="clr-namespace:Syncfusion.Maui.ProgressBar;assembly=Syncfusion.Maui.ProgressBar">
+
+    <progressBar:SfStepProgressBar />
+
+</ContentPage>
+
+{% endhighlight %}
+
+{% highlight C# %}
+
+using Syncfusion.Maui.ProgressBar;
+...
+
+SfStepProgressBar stepProgressBar = new SfStepProgressBar();
+this.Content = stepProgressBar ;
+
+{% endhighlight %}
+
+{% endtabs %}
+</td></tr>
+</table>
+
+<table>
+<tr>
+<th>Xamarin SfStepProgressBar</th>
+<th>.NET MAUI SfStepProgressBar</th>
+<th>Description</th>
+</tr>
+<tr>
+<td>{{'[CompletedStepStyle]()'| markdownify }}</td>
+<td>{{'[CompletedStepSettings]()'| markdownify }}</td>
+<td>Gets or sets the completed step style for the step progressbar.</td>
+</tr>
+<tr>
+<td>{{'[InProgressStepStyle]()'| markdownify }}</td>
+<td>{{'[InProgressStepSettings]()'| markdownify }}</td>
+<td>Gets or sets the in progress step style for the step progressbar.</td>
+</tr>
+<tr>
+<td>{{'[NotStartedStepStyle]()'| markdownify }}</td>
+<td>{{'[NotStartedStepSettings]()'| markdownify }}</td>
+<td>Gets or sets the not started step style for the step progressbar.</td>
+</tr>
```

Comment:
Gets or sets the label position or tile alignment.

Add description about maui property and its use case

Reply:
Added description about the maui API